lfucg-exactions/server/accounts/viewsets.py

```diff
@@ -289,10 +291,39 @@ def create(self, request):
         data_set['created_by'] = self.request.user.id
         data_set['modified_by'] = self.request.user.id
 
-        serializer = AccountLedgerSerializer(data=data_set)
-        if serializer.is_valid(raise_exception=True):
-            self.perform_create(serializer)
-            return Response(serializer.data)
+        if 'plat' in self.request.data:
+            chosen_plat = self.request.data['plat']
+            plat_set = Plat.objects.filter(id=chosen_plat)
+            
+            non_sewer_credits_per_lot = 0
+            sewer_credits_per_lot = 0
+
+            if plat_set is not None:
```

Comment:
an empty queryset is not None. Use plat_set.exists() or plat_set.count() instead